modules/exploits/windows/ftp/quickshare_traversal_write.rb

```diff
@@ -48,6 +48,7 @@ def initialize(info={})
         [
           ['QuickShare File Server 1.2.1', {}]
         ],
+      'Stance' => Msf::Exploit::Stance::Aggressive,
```

Comment:
Override the `Stance` set by `Exploit::Remote::SocketServer` inherited from `Exploit::Remote::TcpServer`